Presentation.Web/Models/API/V2/Internal/Response/Organizations/UIModuleCustomizationResponseDTO.cs

```diff
@@ -7,6 +7,6 @@ public class UIModuleCustomizationResponseDTO
 
         public string Module { get; set; }
 
-        public IEnumerable<CustomizedUINodeResponseDTO> Nodes { get; set; }
+        public IList<CustomizedUINodeResponseDTO> Nodes { get; set; }
```

Comment:
I think IEnumerable is better, it's the more broad type as IList implements IEnumerable